example-workflows/gce/README.md

```diff
@@ -0,0 +1,76 @@
+<!-- Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+# Google Compute Engine - GitHub Actions
+
+An example workflow that uses [GitHub Actions](https://help.github.com/en/categories/automating-your-workflow-with-github-actions) to deploy a container to an existing [Google Compute Engine](https://cloud.google.com/compute-engine/) (GCE) instance.
+
+## Workflow
+
+For pushes to the _default_ branch, `master`, the workflow will:
+
+1. Verify the Google Cloud Platform credentials are correct.
+1. Build, tag, and push the image to Google Container Registry.
+1. Deploy the container image to a Google Compute Engine instance.
+    * Note that a GCE deployment requires an existing [container-optimized VM][create-vm].
+
+## Setup
+
+1. Create or reuse a Github repository for the example workflow:
```

Comment:
Was there meant to be a link after "workflow:"?

Reply:
The following nested bulletin item list is what it's referring to.